### PR TITLE
MOB-78 call question api endpoint from webapp

### PR DESCRIPTION
**Pull request by lukasrakauskas**

### CODE REVIEW

File: src/environments/environment.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 export const environment = {
   production: false,
-  apiEndpoint: 'https://localhost:44378/api/v1'
+  apiEndpoint: 'http://localhost:44378/api/v1',
```
Comment (Dezash) [inline src/environments/environment.ts]: IIS has SSL enabled, what is the reason of changing it to http?

Comment (lukasrakauskas) [reply src/environments/environment.ts]: WebApp in dev environment doesn't run with SSL enabled thus causes errors

Comment (lukasrakauskas) [reply src/environments/environment.ts]: Fixed

File: src/environments/environment.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 export const environment = {
   production: false,
-  apiEndpoint: 'https://localhost:44378/api/v1'
+  apiEndpoint: 'https://localhost:44378/api/v1',
```
Comment (wizacass) [inline src/environments/environment.ts]: Haven't we decided to use port 5000 for the API?

Comment (Dezash) [reply src/environments/environment.ts]: @wizacass if it's going to be https, it has to be 5001

Comment (lukasrakauskas) [reply src/environments/environment.ts]: Done

Comment (wizacass) [reply src/environments/environment.ts]: Lets leave it as http for now, as we might have problems with self-signed certificates on localhost. And once we do production release, then lets switch to https.

**Review by wizacass — APPROVED** [review]

**Review by MPilinkus — APPROVED** [review]

**Pull request merged**